lib/static_sync/processor.rb

```diff
@@ -46,6 +46,8 @@ def handle_conflict(file)
         if file.cached?
           raise ConflictError, "modifications to existing cached files are not allowed."
```

Comment:
It's matching because we are setting this flag https://github.com/compactcode/static_sync/blob/1b94905c4ddd12bc0d12dac9e4e060317e5bd277/lib/static_sync/config.rb#L43